packages/dialog/src/vaadin-dialog-base-mixin.js

```diff
@@ -137,6 +165,13 @@ export const DialogBaseMixin = (superClass) =>
       }
     }
 
+    __positionChanged(top, left) {
+      if (top == null || left == null) {
+        return;
+      }
+      this.$.overlay.setBounds({ top, left });
```

Comment:
Looks like setting `top` and `left` using attributes in HTML isn't supported with the current implementation as `setBounds()` checks for `typeof parsedBounds[arg] === 'number'` so the following is ignored:

```html
<vaadin-dialog top="100" left="50"></vaadin-dialog>
```

Maybe we could use something like `!isNaN(parsedBounds[arg])` which returns `true` for parsable strings.

Reply:
Good catch! Done.